composer.json

```diff
@@ -28,11 +28,11 @@
 		}
 	},
 	"require" : {
-		"php" : ">=5.6",
+		"php" : ">=7.2",
 		"symfony/polyfill-mbstring" : "~1.8"
 	},
 	"require-dev" : {
-		"phpunit/phpunit" : "^5.7",
+		"phpunit/phpunit" : ">=7.0",
```

Comment:
Consider defining the PHPUnit `^8.0` version at least because the `PHPUnit ^7.0` will support `php-7.1`.

But I see that the `php-7.1` version will be dropped from `.travis.yml` setting.

Reply:
agree

Reply:
Done